app/presenters/screen_reader_filter_description_presenter.rb

```diff
@@ -0,0 +1,48 @@
+class ScreenReaderFilterDescriptionPresenter
+  def initialize(filters, sort_option)
+    @filters = filters
+    @sort_options = sort_option.present? ? ["sorted by #{sort_option['name']}"] : []
+  end
+
+  def present
+    text = (facets_without_tags + sort_options).compact.join(", ")
+    "<span class='visually-hidden'>#{text}</span>"
+  end
+
+private
+
+  attr_reader :filters, :sort_options
+
+  def facets_without_tags
+    description = filters.each_with_object([]) do |filter, facets_description|
+      if filter.hide_facet_tag?
```

Comment:
I'd suggest a `reject` before this. That way you don't need the `compact` call later.